package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "d3": "^3.5.17",
     "lodash": "^4.13.1",
+    "mark.js": "github:julmot/mark.js#649fdcd",
```

Comment:
Locking down a dependency to a specific version until you explicit want to update isn't that bad:
- The commit-ish syntax is also used in the "plottable" dependency
- The "normalize.css" dependency is also fixed
- Automatically updating libraries (even minor or patch releases) may break you application as there is no guarantee that there aren't any issues with your exact use case. Therefore it's always safer to just update a library if you have tested it yourself and this might not be the case with automatic updates. Imagine the situation: You're updating something on the layout and automatically updating all dependencies. But you haven't tested the whole site, just your simple change. Then it could happen that something else stops working.

But, the chance to break something with automatic non-major updates is low, so I'm on your side. However, as I said, I don't see any way to express `^7.0.0` with the GitHub syntax of npm.
If you have a better idea let me know :blush: